.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
-    "python.testing.pytestArgs": [],
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
```

Comment:
Was this causing a problem?

Reply:
yes, every time on opening this needed to be configured